### ISSUE TITLE
Port integration with Promotheus from amq collection

### ISSUE THREAD

**Issue by rpelisse** [ansible-middleware/amq_streams#11]
##### SUMMARY

The  [amq collection has support for Promotheus](https://github.com/search?q=repo%3Aansible-middleware%2Famq%20prometheus&type=code) that could be easily (hopefully) ported into this one.

##### ISSUE TYPE
 - Feature Idea

**Comment by RanabirChakraborty** [comment]
@rpelisse can I work on it?

**Comment by rpelisse** [comment]
@RanabirChakraborty Please do!

**Comment by rpelisse** [comment]
@RanabirChakraborty Are you anywhere on this topic? if you need any help, maybe ask Guido or @rmarting?

**Comment by rmarting** [comment]
AFAIK Prometheus export agent is not included in AMQ Streams for RHEL. The monitoring can be done using JMX ports as another JVM process. The documentation describing that is here: https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.4/html/using_amq_streams_on_rhel/monitoring-str#con-jmx-configuration-options-str

**Comment by rpelisse** [comment]
@rmarting Given your feedback and the use of JMX instead of Promotheus, I wonder if we should not simply close this. If someone wants to provide integration with Promotheus, we'll be happy to merge it, but I don't think it should be on our priority list. WDYT?

**Comment by rmarting** [comment]
I agree @rpelisse ! The latest commits allow to declare the JMX properties so they will allow to monitor and get the metrics from Kafka components. As there is no an official way to implement the Prometheus integration, we can close this issue. In the future, a new one can track better that integration.

Go ahead and close it!

**Comment by rpelisse** [comment]
Fixed by https://github.com/ansible-middleware/amq_streams/pull/93
